README.md

```diff
@@ -6,6 +6,17 @@
 [![Release version](https://img.shields.io/github/release/pmem/vmem.svg?sort=semver)](https://github.com/pmem/vmem/releases/latest)
 [![Coverage Status](https://codecov.io/github/pmem/vmem/coverage.svg?branch=master)](https://codecov.io/gh/pmem/vmem/branch/master)
 
+## ⚠️ Discontinuation of the project
+The **ibvmem** and **libvmmalloc** project will no longer be maintained by Intel.
```

Comment:
hmm, actually `libvmem and libvmmalloc project` is an unfortunate sentence 😉 perhaps we could say:

1. libvmem project will no...
or
2. libvmem and libvmmalloc libraries will no...

Reply:
done